Directory.Build.targets

```diff
@@ -35,6 +35,7 @@
     <PackageReference Update="Moryx.Runtime.Maintenance" Version="$(MoryxCoreVersion)" />
     <PackageReference Update="Moryx.Runtime.Kernel" Version="$(MoryxCoreVersion)" />
     <PackageReference Update="Moryx.Runtime.Kestrel" Version="$(MoryxCoreVersion)" />
+    <PackageReference Update="Moryx.Asp.Extensions" Version="$(MoryxCoreVersion)" />
```

Comment:
Would you add the controller related package references here as well? We will need them for the resourcemanagement as well 😊